use_case_examples/lora_finetuning/data_finetune/raw_cml_1.7.0_examples.txt

```diff
@@ -0,0 +1,458 @@
+**1. Linear Models:**
```

Comment:
what's the purpose of this file ?

Reply:
Raw training dataset. I have a script that convert this into a proper dataset.json